bin/commands/disable.js

```diff
@@ -6,6 +6,7 @@ var
 module.exports = function () {
 
   var kuzzle = new Kuzzle(false);
-  kuzzle.enable(params, true);
+
+  kuzzle.remoteActions(kuzzle, 'enableServices', params, {enable: false});
```

Comment:
I am a bit disturbed by passing the `kuzzle` object to itself down to the final action.

Could Kuzzle constructor pass itself to the remoteActions?